llvm/tools/sycl-post-link/SpecConstants.cpp

```diff
@@ -215,6 +219,18 @@ std::string mangleFuncItanium(StringRef BaseName, const FunctionType *FT) {
   return Res;
 }
 
+MDNode *generateSpecConstDefaultValueMetadata(Instruction *I, StringRef SymID,
+                                              Value *Default) {
+  LLVMContext &Ctx = I->getContext();
+  SmallVector<Metadata *, 4> MDOperands;
+  // MDOperands.push_back(MDString::get(Ctx, SymID));
+  MDOperands.push_back(ConstantAsMetadata::get(cast<Constant>(Default)));
+
+  // MDNode *Entry = MDNode::get(Ctx, MDOperands);
+  return MDNode::get(Ctx, MDOperands);
```

Comment:
IIRC, the second argument here is `ArrayRef`, which has a constructor that accepts a reference to a single argument - we can use it here, I guess to simplify this function